source/features/list-prs-for-file.tsx

```diff
@@ -131,7 +134,7 @@ async function init(): Promise<void> {
 						: (
 							<>
 								Careful, {prs.length} open PRs are already touching this file
-								<span className="ml-2 BtnGroup" style={{verticalAlign: '-0.6em'}}>
+								<span className="ml-2 BtnGroup">
```

Comment:
This is added more than 1.5 years ago: [`3ead08d` (#2870)](https://github.com/sindresorhus/refined-github/pull/2870/commits/3ead08dfbebe5fbd77f1d1738825f5938e4c9374#diff-26e829c321ccba564beb4bb33adb6e5a71106315a1e35b430d5dd1d7b383aa05R68)

Seems no longer needed